core/unit_test/TestGraph.hpp

```diff
@@ -127,7 +127,56 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), launch_one_rvalue) {
   ASSERT_EQ(0, bugs_host());
 }
 
-TEST_F(TEST_CATEGORY_FIXTURE(graph), launch_six) {
+/// @test Ensure that @c Kokkos::Graph::instantiate works.
+/// @note For now, @c Kokkos::Graph::submit will instantiate if needed,
+/// so this test is not very strong.
+TEST_F(TEST_CATEGORY_FIXTURE(graph), instantiate_and_submit_once) {
+  auto graph = Kokkos::Experimental::create_graph(ex, [&](auto root) {
+    root.then_parallel_for(1, count_functor{count, bugs, 0, 0});
+  });
+  graph.instantiate();
+  graph.submit();
+  Kokkos::deep_copy(ex, count_host, count);
+  Kokkos::deep_copy(ex, bugs_host, bugs);
+  ex.fence();
+  ASSERT_EQ(1, count_host());
+  ASSERT_EQ(0, bugs_host());
+}
+
+#define GTEST_SKIP_IF_NO_ASSERTION(__msg__)   \
```

Comment:
Just like our user should abstain to define KOKKOS_ macros, we should probably not be defining GTEST_ones.
Also there is little point defining anything as we only use it once.

Reply:
I'll revert this change, so this thread does not make sense anymore. Closing.